packages/driver/src/cypress/commands.ts

```diff
@@ -255,16 +254,9 @@ export default {
         })
       },
 
-      addChainer (obj) {
-        // perp loop
-        for (let name in obj) {
-          const fn = obj[name]
-
-          cy.addChainer(name, fn)
-        }
-
-        // prevent loop comprehension
-        return null
+      addSelector (name, fn) {
+        // TODO: Add overriding stuff.
+        return cy.addSelector(name, fn)
```

Comment:
Ah, whoops, slight code-splatter here, had meant to remove this before committing. I'll clean it up in an upcoming refactor. Not used yet, shouldn't be in yet. Same with `selectorFns` below.